osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderTail.cs

```diff
@@ -125,8 +126,15 @@ protected override void UpdateHitStateTransforms(ArmedState state)
 
         protected override void CheckForResult(bool userTriggered, double timeOffset)
         {
-            if (!userTriggered && timeOffset >= 0)
-                ApplyResult(r => r.Type = Tracking ? r.Judgement.MaxResult : r.Judgement.MinResult);
+            if (userTriggered)
+                return;
+
+            // The player needs to have engaged in tracking at any point after the tail leniency cutoff.
+            // An actual tick miss should only occur if reaching the tick itself.
+            if (timeOffset >= SliderEventGenerator.TAIL_LENIENCY && Tracking)
```

Comment:
One thing I didn't realise earlier is that this isn't 100% correct. If a slider is shorter than 72ms, the tail leniency actually ends midway through the slider, rather than a constant 36ms back, as per:

```csharp
            double finalSpanEndTime = Math.Max(startTime + totalDuration / 2, (finalSpanStartTime + spanDuration) + TAIL_LENIENCY);
```

in `LegacyLastTick` generation.

Reply:
Yeah, I noticed this one but forgot to reapply it here.

Reply:
Honestly not sure we want to apply this? If we're changing the behaviour, it seems more correct to remove this completely..

Reply:
I suppose. Probably still need a check for sliders shorter than 36ms just to avoid any weirdness with slider tails getting judged before the actual slider starts.

Reply:
@peppy is that "shorter than 36ms" check happening here or are we leaving it be as irrelevant? Just wanna make sure this hasn't been forgotten.

Reply:
Actually I guess the "check all preceding nested objects judged" thing covers that since, among others, the slider head is also a nested, so the tail won't be judged before the head is...